packages/knip/test/plugins/simple-git-hooks.test.ts

```diff
@@ -0,0 +1,23 @@
+import { test } from 'bun:test';
+import assert from 'node:assert/strict';
+import { main } from '../../src/index.js';
+import { resolve } from '../../src/util/path.js';
+import baseArguments from '../helpers/baseArguments.js';
+import baseCounters from '../helpers/baseCounters.js';
+
+const cwd = resolve('fixtures/plugins/simple-git-hooks');
+
+test('Find dependencies with the simple-git-hooks plugin', async () => {
+  const { /* issues, */ counters } = await main({
+    ...baseArguments,
+    cwd,
+  });
+
+  // console.log(issues);
+
+  assert.deepEqual(counters, {
+    ...baseCounters,
+    binaries: 2,
+    devDependencies: 2,
```

Comment:
Any chance you could add assertions about what binaries and devDependencies are considered issues here? E.g. `assert(issues.binaries['file']['executable']`?

Reply:
Updated, thanks for your review.